### DISCUSSION TITLE
using light functions on spike prime hub

### DISCUSSION THREAD

**Discussion by crosszay1** [pybricks/support#1871] [Q&A]
I am working on a remote control program and I successfully got it to work! anyways it is for a sumo bot and I was hoping to add a little face on the spike prime screen. I read the docs for light over at https://docs.pybricks.com/en/stable/hubs/primehub.html but I can't seem to find where display is stored! (I am not a python coder but basically I can't find what to put after the "from" in import thingie) any help would be greatly appreciated!

please excuse my lack of knowledge for python terms

**Comment by dlech** [comment]
It isn't a separate import, it is part of the `hub` object. 

There are some examples of how to use it further down that page: https://docs.pybricks.com/en/stable/hubs/primehub.html#matrix-display-examples
